src/workerd/api/node/node.h

```diff
@@ -101,6 +105,10 @@ void registerNodeJsCompatModules(Registry& registry, auto featureFlags) {
           featureFlags.getWorkerdExperimental();
     }
 
+    if (isNodeOsModule(module.getName())) {
+      return featureFlags.getEnableNodeJsOsModule() && featureFlags.getWorkerdExperimental();
```

Comment:
Do we need the experimental check here?
We already got the experimental check on enabling the gate

Reply:
Staying consistent with the other checks in this block. I expect this will be removed very soon, there won't be much reason to keep this experimental for long.